distributed/protocol/pickle.py

```diff
@@ -23,36 +24,46 @@ def _always_use_pickle_for(x):
         return False
 
 
-def dumps(x):
+def dumps(x, *, buffer_callback=None):
     """ Manage between cloudpickle and pickle
 
     1.  Try pickle
     2.  If it is short then check if it contains __main__
     3.  If it is long, then first check type, then check __main__
     """
+    buffers = []
+    dump_kwargs = {"protocol": HIGHEST_PROTOCOL}
+    if HIGHEST_PROTOCOL >= 5 and buffer_callback is not None:
+        dump_kwargs["buffer_callback"] = buffers.append
     try:
-        result = pickle.dumps(x, protocol=pickle.HIGHEST_PROTOCOL)
+        del buffers[:]
```

Comment:
This isn't needed here.

Reply:
Yeah I was worried people might add lines before this and forget to add the deletion. Though maybe changes here are rare. No strong feelings either way. Can drop if you prefer 🙂

Reply:
That's a fair point, fine keeping it. I do find `buffers.clear()` easier to read than `del buffers[:]`, so if you'd be up for changing that, that'd be my preference. Up to you I guess.

Reply:
Sounds good to me. Pushed a commit using `.clear()` instead. Please let me know if that looks ok.